test-resources/java/insert_field_and_initializer/configurations/rules.toml

```diff
@@ -42,7 +42,6 @@ query = """
 """
 replace_node = "pkg_dcl"
 replace = "@pkg_dcl \n import java.util.List;"
-groups = ["Cleanup Rule"]
```

Comment:
Is this a seed rule? Doesn't seem to be setting `is_seed_rule = false`, but it was in the cleanup rules group before...

Reply:
Great catch. I am disappointed I missed this :| 
Anyways, what is shocking is that even without marking "Cleanup Rules", the tests passed. 
I have updated the tests, and added `is_seed_rule= False`

https://github.com/uber/piranha/pull/379/commits/3c93d040785a68836b84f167de336c75056d9578 - Fixed rust tests 
https://github.com/uber/piranha/pull/380/commits/415d23888d6a78bf5b6c774da1f882e10e530ac3 - Fixed PyTest 

At a high level, I added a file that shud not be transformed by the given rule graph.

Reply:
> Anyways, what is shocking is that even without marking "Cleanup Rules", the tests passed.

Do we have an explanation as to why? Might be worth making sure we are not missing a test case there.